UI/admin.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <title>Fast-Food-Fast</title>
+        <meta charset = "UTF-8">
+        <meta name="keywords" lang="en-us" content="events, plot, night life">
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalabe=no">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1">
+        <meta name="HandheldFriendly" content="true">
+        <link rel="stylesheet" type="text/css" href="../UI/CSS/admin.css">
+        <script type="text/javascript" src="js/scripts.js"></script>
+    </head>
+
+    <body>
+
+        <style>
```

Comment:
You could put this in your css file and define an id for the body tag to do away with the inline css.